crates/ruma-client-api/src/error.rs

```diff
@@ -166,6 +166,13 @@ pub enum ErrorKind {
     /// The desired user name is not valid.
     InvalidUsername,
 
+    /// `M_INVITE_BLOCKED`
+    ///
+    /// The invite was interdicted by moderation tools or configured access controls without having
+    /// been witnessed by the user.
```

Comment:
It seems clearer to me to specify which user we are talking about.

```suggestion
    /// been witnessed by the invitee.
```